packages/core/src/modules/vc/data-integrity/W3cJsonLdCredentialService.ts

```diff
@@ -265,6 +265,14 @@ export class W3cJsonLdCredentialService {
         challenge: options.challenge,
         domain: options.domain,
         documentLoader: this.w3cCredentialsModuleConfig.documentLoader(agentContext),
+        checkStatus: async ({ credential }: { credential: W3cJsonCredential }) => {
+          if ('credentialStatus' in credential) {
+            await verifyBitStringCredentialStatus(credential, agentContext)
+          }
```

Comment:
verifyPresentation also has options.verifyCredentialStatus, can you add that check here as well?